lib/vendor/PEAR/Installer.php

```diff
@@ -1524,7 +1524,7 @@ function sortPackagesForUninstall(&$packages)
         if (PEAR::isError($this->_dependencyDB)) {
             return $this->_dependencyDB;
         }
-        usort($packages, array(&$this, '_sortUninstall'));
+        usort($packages, array($this, '_sortUninstall'));
     }
 
     function _sortUninstall($a, $b)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 6, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)